### DISCUSSION TITLE
how ??

### DISCUSSION THREAD

**Discussion by AnandPatel404** [devlikeapro/waha#310] [General]
behind the scene you use baileys what the new in this package i use baileys more then two years baileys provide more features then you then why i choose you ??

**Comment by allburov** [comment]
Well, there's few reasons why **people** choose the project:
1.Single API for WebjS, Balieys, Venom (deprecated one, but yet). We can add many engines here without people having to adjust application code (almost)
2. HTTP API Requests, that means you don't have to use and deal with JS\TS\nodejs ever, you can use python/c# or other language.
3. HTTP Webhooks - the same as 2, but for receiving messages.
4. Built-in Security for API and webhooks
5. Docker easy one step installation
6. May be some other reason, tutorial, video, IDK.

Sure, **no reason** if you're well experienced with baileys and can build HTTP API yourself! :+1: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by AnandPatel404** [reply]
👍✅
